reference_data/management/commands/update_gencode_latest.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from reference_data.management.commands.utils.gencode_utils import load_gencode_records, create_transcript_info, \
+    LATEST_GENCODE_RELEASE
+from reference_data.management.commands.utils.update_utils import update_records
+from reference_data.management.commands.update_refseq import RefseqReferenceDataHandler
+from reference_data.models import GeneInfo, TranscriptInfo
+
+logger = logging.getLogger(__name__)
+
+BATCH_SIZE = 5000
+
+
+class Command(BaseCommand):
+    help = 'Loads genes and transcripts from the latest supported Gencode release, updating previously loaded gencode data'
```

Comment:
I see the differences between `update_gencode` and `update_gencode_latest`. But it can be confusing for the users. How about add a scenario description about when to use which command?